### ISSUE TITLE
Drag and drop doesn't work on secondary monitor.

### ISSUE THREAD

**Issue by blaqsirens** [cdb-boop/ComfyUI-Model-Manager#2]
I'm using firefox on my secondary monitor and when I try to drag and drop a model or lora onto the graph it simply doesn't work. It works perfectly fine on whichever monitor is marked as the main display in windows display settings. 

In Chromium based browsers this doesn't seem to be an issue, although those browsers all seemed to have an issue where drag and dropping a model or lora caused ComfyUI to lock up for about 5-10 seconds until the node is created. Firefox adds nodes instantly when being used on the main display.

My issue is mainly with Firefox though as ComfyUI simply works better on it. Other than that everything works great, although it would be nice if it loaded into sidebar mode on the models tab, or had an option to configure that. Thanks for picking up this up and continuing to improve it!

**Comment by cdb-boop** [comment]
Yes, drag and drop in Firefox isn't working at all when visible on a secondary (right) monitor. The add button works fine and there is no issue in Chrome. It looks like an issue with `document.elementFromPoint` and how Firefox handles xy coordinates. I will try to fix it.

However, I didn't encounter any lock up.

**Comment by blaqsirens** [comment]
Thanks for looking into this issue. I haven't tried Chrome yet because the last time I had Chrome installed, it would scan through my computer's hard drives and slow down my system, so I've been trying to avoid it. 

The browsers I tried were Chromium-based ones like Brave and Edge, and on top of the UI lock up when dragging and dropping, the UI was overall a lot more sluggish than any of the Firefox based browsers I tried, so this may be an issue with my system.
